samples/Sample.Forms/Samples/BindingPageViewModel.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+using SignaturePad.Forms;
+
+namespace Samples
+{
+	public class BindingViewModel : BindableObject
+	{
+		private IEnumerable<Point> currentSignature;
+		private Point[] savedSignature;
+
+		public BindingViewModel (Func<SignatureImageFormat, ImageConstructionSettings, Task<Stream>> getImageDelegate)
+		{
+			GetImageStreamAsync = getImageDelegate;
+
+			SaveVectorCommand = new Command (OnSaveVector);
+			LoadVectorCommand = new Command (OnLoadVector);
+			SaveImageCommand = new Command (OnSaveImage);
+		}
+
+		public IEnumerable<Point> CurrentSignature
+		{
+			get => currentSignature;
+			set
+			{
+				currentSignature = value;
+				OnPropertyChanged ();
+			}
+		}
+
+		public Point[] SavedSignature
+		{
+			get => savedSignature;
+			set
+			{
+				savedSignature = value;
+				OnPropertyChanged ();
+				OnPropertyChanged (nameof (HasSavedSignature));
```

Comment:
I think it's redundant, because HasSavedSignature should auto recalculate as it's depend from SavedSignature

Reply:
Unfortunately bindings are not transitive like this.